src/libs/SearchUIUtils.ts

```diff
@@ -488,7 +491,8 @@ function getTaskSections(data: OnyxTypes.SearchResults['data']): TaskListItemTyp
             }
 
             return result;
-        });
+        })
+        .filter(isTaskListItemType);
```

Comment:
```suggestion
        })
```
Lets remove this, that was some testing that I pushed